cinempc/DroneAndCameraState drone_state
cinempc/TargetState[] targets_state
int8 targets_found

